src/libraries/System.Runtime.InteropServices/tests/DllImportGenerator.UnitTests/TestUtils.cs

```diff
@@ -89,37 +123,54 @@ public static async Task<Compilation> CreateCompilation(SyntaxTree[] sources, Ou
         }
 
         /// <summary>
-        /// Create a compilation given source and reference assemblies
+        /// Create a compilation given source and a target framework
         /// </summary>
         /// <param name="source">Source to compile</param>
-        /// <param name="referenceAssemblies">Reference assemblies to include</param>
+        /// <param name="targetFramework">Target framework</param>
         /// <param name="outputKind">Output type</param>
         /// <param name="allowUnsafe">Whether or not use of the unsafe keyword should be allowed</param>
         /// <returns>The resulting compilation</returns>
-        public static Task<Compilation> CreateCompilationWithReferenceAssemblies(string source, ReferenceAssemblies referenceAssemblies, OutputKind outputKind = OutputKind.DynamicallyLinkedLibrary, bool allowUnsafe = true)
+        public static async Task<Compilation> CreateCompilationWithReferenceAssemblies(string source, TestTargetFramework targetFramework, OutputKind outputKind = OutputKind.DynamicallyLinkedLibrary, bool allowUnsafe = true)
```

Comment:
`CreateCompilationForTargetFramework`?

Reply:
Actually, maybe merge this with `CreateCompilation` now with a `targetFramework` default of `TestTargetFramework.Net`?

Reply:
Done.